ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/model/OpenAPIInfo.java

```diff
@@ -49,13 +63,48 @@ public Optional<String> getContractPath() {
         return Optional.ofNullable(this.contractPath);
     }
 
+    public Optional<String> getDescription() {
```

Comment:
Are all these fields are nullable ? If not lets remove `Optional` from unwanted fields.

Reply:
These fields are intended to have null during the value extraction process, therefore we keep those fields enabling the null.